scripts/ftp_deploy.rb

```diff
@@ -0,0 +1,12 @@
+require 'net/sftp'
+
+host = ENV['100DAYS_FTP_HOST']
+username = ENV['100DAYS_FTP_USER']
+password = ENV['100DAYS_FTP_PASS']
+port = ENV['100DAYS_FTP_PORT']
+
+base_dir = "/home/#{username}/100days.envirodatagov.org"
```

Comment:
I don't think this is accurate if we define new ftp users on cPanel? (the path isn't tied to username)

Reply:
You're right! Just investigated cpanel, and created some users with more limited access to 100days dir